requirements.txt

```diff
@@ -3,6 +3,8 @@ scipy>=1.2.3,<2
 scikit-image>=0.14.5
 scikit-learn>=0.20.4
 tensorflow==2.4.1
+tensorflow_addons==0.12.1
+spektral==1.0.4
```

Comment:
These also need to be added to `setup.py`. They both exist due to the pip and docker builds.

Reply:
Added